website/docs/cli/commands/plan.mdx

```diff
@@ -149,15 +149,13 @@ the previous section, are also available with the same meanings on
   it would effectively disable the entirety of the planning operation in that
   case.
 
-* `-replace=ADDRESS` - Instructs Terraform to plan to replace the single
+- `-replace=ADDRESS` - Instructs Terraform to plan to replace the single
   resource instance with the given address. If the given instance would
   normally have caused only an "update" action, or no action at all, then
   Terraform will choose a "replace" action instead.
 
-  You can use this option if you have learned that a particular remote object
-  has become degraded in some way. If you are using immutable infrastructure
-  patterns then you may wish to respond to that by replacing the
-  malfunctioning object with a new object that has the same configuration.
+  You can use this option if you have learned that one or more remote objects
+  have become degraded in some way. Add multiple `-replace` flags to replace several objects at once. If you are using immutable infrastructure patterns, you may wish to replace the malfunctioning objects with new objects that have the same configuration.
```

Comment:
I'm not sure how much this really matters, but FWIW this read a bit weirdly to me since I feel like the comment about immutable infrastructure patterns follows on from the first sentence, but now those two thoughts are split by the unrelated note about using `-replace` multiple times.

Seeing this makes me think that there might be a better phrasing for what was here before to capture the two parts of the thought in a single sentence. I don't yet have concrete suggestion for that, but the intended idea here is that one of the typical stated advantages of "immutable infrastructure" (i.e. that you don't change any internal state of an object after it's created, as might be true for a stateless application server) is that you can react to single-object failures by just replacing that object with a fresh one, instead of trying to repair the existing object in-place.

Given that, `-replace` is _particularly useful_ when employing that approach, and conversely likely to be less useful for mutable stateful objects (like e.g. a database) where the new object would not be equivalent to the one it replaced (you'd lose all of the data that was added to the database since it was created).